lib/index.js

```diff
@@ -1,7 +1,8 @@
 let buildBundler = require("./build-bundler");
 
-module.exports = (pluginConfig, assetManager, { watcher, browsers, compact }) => {
-	let config = buildConfig(pluginConfig, assetManager, { browsers, compact });
+module.exports = (pluginConfig, assetManager, options = {}) => {
```

Comment:
This default value is a change in behaviour unrelated to source maps, isn’t it?  Is that intentional?